model-optimizer/extensions/ops/elementwise.py

```diff
@@ -22,13 +23,49 @@ def override_data_type_of_constant(node: Node):
         # power value 2, or when replacing Neg operation with Mul with -1 as second input.
         in_node_0 = node.in_port(0).get_source().node
         in_node_1 = node.in_port(1).get_source().node
-        if in_node_0.op == 'Const':
-            convert_const_node_value_type(in_node_0, in_type_1)
-        elif in_node_1.op == 'Const':
-            convert_const_node_value_type(in_node_1, in_type_0)
+
+        if in_node_0.soft_get('op') != 'Const' and in_node_1.op != 'Const':
+            raise Exception("Elementwise operation '{}' has inputs of different data types: '{}' and '{}' "
```

Comment:
Use `Error` class to raise the exception. The `Error` class exception is gracefully handled in the MO main class.

Reply:
When there was raised just an `Error` from `mo.utils` I haven't seen a full picture just a message 
![image](https://user-images.githubusercontent.com/5703039/120767782-33979a00-c524-11eb-89d4-0dd2aa783710.png)

Compare with the situation where we have here Exception. I see traceback I have wider understanding where, what, why and at which stage it was gone wrong
![image](https://user-images.githubusercontent.com/5703039/120768109-7e191680-c524-11eb-907b-edb91a6f7cd9.png)

But if there are reasons why Error is better and you insist I am ready to return it back

Reply:
This is exactly our point. Until user specify `\-\-log_level DEBUG` we don't want to bother user with error details. Please, use `Error`.

Reply:
replaced `Exception` -> `Error`